### DISCUSSION TITLE
Delete single log from file

### DISCUSSION THREAD

**Discussion by Chris-Richards** [opcodesio/log-viewer#283] [Ideas]
Speaks for itself

**Comment by arukompas** [comment]
hey @Chris-Richards , I thought about this before as well, but couldn't think of a good reason why someone would want to do this if it wasn't malicious (to delete a couple of log entries without raising suspicion). It would also complicate the whole UI as well, requiring a button on every log to delete it, and then an authorization gate of whether that one particular log can be deleted by someone.

Let me know what you think.

**Reply by Chris-Richards** [reply]
For me personally this would be SUPER useful, I use the native laravel logging heavily for application monitoring (i log a lot of info not just errors) and would benefit significantly from being able to a. individually delete each log line (a button displaying on hover for each log would be fine) and b. being able to delete each log type from the whole file (delete all errors but keep info and all other types ect).
